scripts/pre_push_hook.py

```diff
@@ -274,9 +274,16 @@ def start_linter(files):
 
 def start_python_script(scriptname):
     """Runs the 'start.py' script and returns the returncode of the task."""
+    script_commands_list = scriptname.split(' ')
```

Comment:
Can you please take a look at what calls this function? It's used with various different args, none of which have a '/' in it, so I don't know what's going on with the replacement of '/'. The docstring is also completely wrong. As a result, I have no idea what is going on in this function ... some refactoring might be needed.

If you are keeping this function, I would suggest passing the `cmd` array in, instead -- and renaming the function to run_cmd_and_get_returncode or similar.

Reply:
The `/` is because of the path. Example: `scripts/run_frontend_tests` should be `scripts.run_frontend_tests`. Also, I think it would better to pass the `scriptname` instead because it will avoid repeating the construction of python scripts everytime. What do you think?